### ISSUE TITLE
Feature request: 'Search' on `government.github.com/community` should consider the description, etc of each account

### ISSUE THREAD

**Issue by daveloyall** [github/government.github.com#1261]
This is a feature request.

If the 'search' feature on https://government.github.com/community/ could somehow include data from the "About" field of each account, then search results would be better.

Here is a concrete example..

Search for "nebraska".  You get one result in the "U.S. States" category.  However, there are actually two: the `nelegislature` account should be in these search results.

When you view that account (https://github.com/nelegislature) you see that the word "nebraska" appears in more than one field.  If those fields were included in search, this second account would be included in the results.

**Comment by mlinksva** [comment]
Good idea. It'd probably require storing such metadata in this repo, so it'd be significant change/work. Suggested solutions/approaches welcome.
